src/components/AlbumItem.jsx

```diff
@@ -35,6 +35,9 @@ export class AlbumItem extends Component {
         url: link,
         isLoading: false
       }))
+      .catch(linkError => {
```

Comment:
To improve purity of this method, I think we may move the catch directly in the constructor, after `fetchPhoto` call. What do you think ?